container/navigation.go

```diff
@@ -0,0 +1,196 @@
+package container
+
+import (
+	"fyne.io/fyne/v2"
+	"fyne.io/fyne/v2/layout"
+	"fyne.io/fyne/v2/theme"
+	"fyne.io/fyne/v2/widget"
+)
+
+// Navigation container is used to provide your application with a control bar and an area for content objects.
+// Objects can be any CanvasObject, and only the most recent one will be visible.
+//
+// Since: 2.6
+type Navigation struct {
+	widget.BaseWidget
+
+	Root  fyne.CanvasObject
+	Title string
+	Back  NavigationObject
+	Next  NavigationObject
+	Label *widget.Label
+
+	level  int
+	stack  fyne.Container
+	titles []string
+}
+
+// NavigationObject allows using any object implementing the `fyne.Disableable`,
+// `fyne.CanvasObject`, and `fyne.Tappable` interfaces to be used as Back/Next
+// navigation control.
+//
+// Since: 2.6
+type NavigationObject interface {
+	fyne.CanvasObject
+	fyne.Disableable
+	fyne.Tappable
+}
```

Comment:
Like indicated above, I'm sceptical about this interface and the custom solution for changing the backwards and next buttons. I'd prefer to just have an `OnBack` and an `OnNext` function to keep things simple. I'd rather begin without this and we can add this later if there is enough demand.

Reply:
This is a good point - but is slightly ironic because the "forward nagivation" was added in response to it not being sufficient to be called `Navigation` without it - so we might need a little discussion to agree the scope.

Reply:
Actually maybe I misunderstood - with or without forward navigation this interface should not be needed.

Reply:
I didn't really like it either - now gone.